src/checkbox-table/checkbox-table.tsx

```diff
@@ -33,6 +33,7 @@ interface Props {
     columns: Array<Column>;
     onChange: (data: CheckboxTableData) => void;
     selectAllLabel: string;
+    isEveryCellDisabled?: (columnKey: string, rowKey: string) => boolean;
```

Comment:
This function is invoked for every cell. Therefore, i'd suggest keeping `isDisabledCell`